src/main/java/tuteez/model/ModelManager.java

```diff
@@ -174,6 +174,20 @@ public void removeLastViewedPerson() {
         logger.info("Last viewed person removed");
     }
 
+    @Override
+    public boolean isLastViewPersonAvailable() {
```

Comment:
Is this function used anywhere else? Besides in the `isSamePersonAsPersonOnDisplay` method. If not, and you don' foresee this method being used anywhere else, perhaps you and use method chaining with the `Optional` class in your `isSamePersonAsPersonOnDisplay` function to make the code cleaner